Library/Formula/gnuradio.rb

```diff
@@ -0,0 +1,230 @@
+  end
+
+  # sphinx starts here
+  resource "sphinx" do
+    url "https://pypi.python.org/packages/source/S/Sphinx/Sphinx-1.3.1.tar.gz"
+    sha256 "1a6e5130c2b42d2de301693c299f78cc4bd3501e78b610c08e45efc70e2b5114"
+  end
+
+  resource "sphinx_rtd_theme" do
+    url "https://pypi.python.org/packages/source/s/sphinx_rtd_theme/sphinx_rtd_theme-0.1.7.tar.gz"
+    sha256 "9a490c861f6cf96a0050c29a92d5d1e01eda02ae6f50760ad5c96a327cdf14e8"
+  end
+
+  resource "docutils" do
+    url "https://pypi.python.org/packages/source/d/docutils/docutils-0.12.tar.gz"
+    sha256 "c7db717810ab6965f66c8cf0398a98c9d8df982da39b4cd7f162911eb89596fa"
+  end
+
+  resource "pygments" do
+    url "https://pypi.python.org/packages/source/P/Pygments/Pygments-2.0.2.tar.gz"
+    sha256 "7320919084e6dac8f4540638a46447a3bd730fca172afc17d2c03eed22cf4f51"
+  end
+
+  resource "jinja2" do
+    url "https://pypi.python.org/packages/source/J/Jinja2/Jinja2-2.7.3.tar.gz"
+    sha256 "2e24ac5d004db5714976a04ac0e80c6df6e47e98c354cb2c0d82f8879d4f8fdb"
+  end
+
+  resource "markupsafe" do
+    url "https://pypi.python.org/packages/source/M/MarkupSafe/MarkupSafe-0.23.tar.gz"
+    sha256 "a4ec1aff59b95a14b45eb2e23761a0179e98319da5a7eb76b56ea8cdc7b871c3"
+  end
+
+  resource "alabaster" do
+    url "https://pypi.python.org/packages/source/a/alabaster/alabaster-0.7.3.tar.gz"
+    sha256 "0703c1ea5a6af0bb6d0cec24708301334949d56ebc7f95c64028d9c66f9d8d5d"
+  end
+
+  resource "babel" do
+    url "https://pypi.python.org/packages/source/B/Babel/Babel-1.3.tar.gz"
+    sha256 "9f02d0357184de1f093c10012b52e7454a1008be6a5c185ab7a3307aceb1d12e"
+  end
+
+  resource "snowballstemmer" do
+    url "https://pypi.python.org/packages/source/s/snowballstemmer/snowballstemmer-1.2.0.tar.gz"
+    sha256 "6d54f350e7a0e48903a4e3b6b2cabd1b43e23765fbc975065402893692954191"
+  end
+
+  resource "six" do
+    url "https://pypi.python.org/packages/source/s/six/six-1.9.0.tar.gz"
+    sha256 "e24052411fc4fbd1f672635537c3fc2330d9481b18c0317695b46259512c91d5"
+  end
+
+  resource "pytz" do
+    url "https://pypi.python.org/packages/source/p/pytz/pytz-2015.2.tar.bz2"
+    sha256 "3e15b416c9a2039c1a51208b2cd3bb4ffd796cd19e601b1d2657afcb77c3dc90"
+  end
+  # sphinx ends here
+
+  resource "cppzmq" do
+    url "https://github.com/zeromq/cppzmq/raw/34c8e4395c94d34a89bbeaaf2b8f9c94a8293c84/zmq.hpp"
+    sha256 "6cdd9d920f4a0f9f3a3257541321bef8369d9735c88b84ba8ae0e461f53e476c"
+  end
+
+  option "with-python", "Build python support"
+  option "with-sphinx", "Build sphinx document"
+  option "with-wavelet", "Build gr-wavelet"
+  option "with-wx", "Build wx gui (requires --with-python)"
+  option "with-qt", "Build qt gui (requires --with-python)"
+  option "with-zeromq", "BUild gr-zeromq"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "swig" => :build if build.with? "python"
+
+  depends_on "boost"
+  depends_on "cppunit"
+  depends_on "fftw"
+  depends_on "python"
```

Comment:
We almost never use unconditional Homebrew Python dependencies because it means when users to try install the bottle they get a Homebrew Python forced upon them, which may not be desired. If it needs 2.7 or above you can do:

```ruby
depends_on :python if MacOS.version <= :snow_leopard
```

If it can use 2.6 you can remove the dependency entirely.